.github/workflows/nightly.yml

```
@@ -0,0 +1,104 @@
+  # §20.4: a million games, five million sampled rows, and a peak-RSS gate.
+  # Minutes to hours — never a merge gate.
+  #
+  # Disabled for now: every test body in tests/load.rs is `todo!()` (the
+  # writer actor and lab runner they exercise don't exist yet), so this job
+  # would fail on every scheduled run rather than reporting a real load-test
+  # result. Flip `if: false` off once those seams are implemented.
+  load:
+    name: load and volume (§20.4)
+    runs-on: ubuntu-latest
+    timeout-minutes: 180
+    if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)